modules/migration-mtc-cr-manifests.adoc

```diff
@@ -0,0 +1,363 @@
+<1> `createDestinationNamespaces`: Creates namespaces for the PVs on the destination cluster if `true`.
+<2> `deleteProgressReportingCRs`: Deletes `DirectVolumeMigrationProgress` CRs if `true`. The default value is `false` so that `DirectVolumeMigrationProgress` CRs are retained for troubleshooting.
+<3> `host`: Update the destination cluster name if necessary.
+<4> Specify one or more PVCs to be migrated with direct volume migration.
+<5> Specify the namespace of each PVC.
+<6> Specify the `MigCluster` CR name of the source cluster.
+
+[id="directvolumemigrationprogress_{context}"]
+== `DirectVolumeMigrationProgress`
+
+`DirectVolumeMigrationProgress` shows the progress of the `DirectVolumeMigration` CR.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: DirectVolumeMigrationProgress
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: <directvolumemigrationprogress_name>
+spec:
+  clusterRef:
+    name: <source_cluster> <1>
+    namespace: openshift-migration
+  podRef:
+    name: <rsync_pod> <2>
+    namespace: openshift-migration
+----
+<1> Cluster on which the `rsync` client pods are running.
+<2> `rsync` pod name.
+
+[id="miganalytic_{context}"]
+== `MigAnalytic`
+
+`MigAnalytic` collects the number of images, Kubernetes resources, and the PV capacity.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigAnalytic
+metadata:
+  annotations:
+    migplan: <migplan_name> <1>
+  name: <miganalytic_name>
+  namespace: openshift-migration
+  labels:
+    migplan: <migplan_name> <2>
+spec:
+  analyzeImageCount: true <3>
+  analyzeK8SResources: true <4>
+  analyzePVCapacity: true <5>
+  listImages: false <6>
+  listImagesLimit: 50 <7>
+  migPlanRef:
+    name: <migplan_name>
+    namespace: openshift-migration
+----
+<1> Specify the `MigPlan` CR name associated with the `MigAnalytic` CR.
+<2> Specify the `MigPlan` CR name associated with the `MigAnalytic` CR.
+<3> `analyzeImageCount`: Optional. Returns the number of images if `true`.
+<4> `analyzeK8SResources`: Optional. Returns the number, kind, and API version of the Kubernetes resources if `true`.
+<5> `analyzePVCapacity`: Optional. Returns the PV capacity if `true`.
+<6> `listImages`: Returns a list of image names if `true`. Default is `false` so that the output is not excessively long.
+<7> `listImagesLimit`: Optional. Specify the maximum number of image names to return if `listImages` is `true`.
+
+[id="migcluster_{context}"]
+== `MigCluster`
+
+`MigCluster` defines a `host`, `local`, or `remote` cluster.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: host <1>
+  namespace: openshift-migration
+spec:
+  isHostCluster: <true> <2>
+  azureResourceGroup: <azure_resource_group> <3>
+  caBundle: <ca_bundle_base64> <4>
+  insecure: false <5>
+  refresh: false <6>
+# The 'restartRestic' parameter is relevant for a source cluster.
```

Comment:
Do we want to include these commented out lines?

Reply:
Yes